src/routes/Login/index.jsx

```diff
@@ -1,11 +1,64 @@
 import React from "react";
-import Navigation from "../../components/Navigation";
+import styles from "./login.module.css";
+
+import EmptySmallEllipseSvg from "../../../public/assets/Ellipse 22.svg";
+import SolidPinkEllipseSvg from "../../../public/assets/Ellipse 19 pink.svg";
+
+import Button from "../../components/Button";
+import Form from "./Form/Form.index";
+
+const {
+  container,
+  top,
+  EmptySmallEllipsePositionOne,
+  EmptySmallEllipsePositionTwo,
+  transparentSvg,
+  SolidPinkEllipsePosition,
+  header,
+  overlay,
+  loginHeader,
+  formMargin,
+  passcodeLink,
+  createAccountLink,
+} = styles;
 
 export default function Login() {
   return (
     <div>
-      <Navigation />
-      <div>This is the login page</div>
+      <div className={container}>
+        <section className={top}>
+          <EmptySmallEllipseSvg className={EmptySmallEllipsePositionOne} />
+
+          <EmptySmallEllipseSvg className={EmptySmallEllipsePositionTwo} />
+
+          <div>
+            <SolidPinkEllipseSvg className={SolidPinkEllipsePosition} />
+            <div className={transparentSvg} />
+          </div>
+
+          <h1 className={header}>Welcome Back</h1>
+        </section>
+
+        <section className={overlay}>
+          <h3 className={loginHeader}>Login</h3>
+
+          <form>
+            <div className={formMargin}>
+              <Form type="email">Email</Form>
+            </div>
+            <Form type="password">Password</Form>
+          </form>
```

Comment:
I think it makes more sense to make component LoginForm, and inside of that component render 2 FormInput component. I love your idea with children inside of Form component, but in your case it is not Form component, it is more FormInput. Also I don't like a div inside of that form only to make a margin, I think this FormInputs can be reusable so maybe just make a margin to each input? I just think that we could have problem if we at some point would like to add a new field?